build-aux/m4/bitcoin_qt.m4

```diff
@@ -112,13 +112,11 @@ AC_DEFUN([BITCOIN_QT_CONFIGURE],[
     if test x$bitcoin_cv_static_qt = xyes; then
       AC_DEFINE(QT_STATICPLUGIN, 1, [Define this symbol if qt plugins are static])
       if test x$qt_plugin_path != x; then
-        QT_LIBS="$QT_LIBS -L$qt_plugin_path/accessible"
```

Comment:
The plugin no longer exists; it was moved to the base library. theuni wants all this conditional though, to support static builds with either version.